contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -568,7 +613,7 @@ contract RaidenMicroTransferChannels {
         delete closing_requests[key];
 
         // Send _balance to the receiver, as it is always <= deposit
-        require(token.transfer(_receiver_address, _balance));
+        require(token.transfer(_receiver_address, _balance - withdrawn_balances[key]));
```

Comment:
Maybe (if not increase gas cost) it'd be nice to also add `_balance - withdrawn_balances[key]` to the `ChannelSettled` event, to inform the actual final transfer value besides the closing/settlement balance?

Reply:
When testing with populus, there is an additional `301` gas cost for cooperativeClose. So, not that much. 
Adding this would actually give enough information for getting the state of the channel when settling, so I'm adding it to the event.